README.md

```diff
@@ -36,100 +37,45 @@ npm install
+## New Token Methods
 
-- actionsDAO: The amount of votes in % from 0-100 needed to excute calls with the DAO approval. Every action is indexed by the address and function signature, they can internal or external actions. To add or edit this actions it will need a DAO approval.
+Líf token is ERC20 compatible but it also has two more methods to allow the transfer of data and execution of calls between users/contracts.
 
 ### transferData
 
 Transfer tokens from one address to another.
 ```
-transferData(address to, uint value, bytes data, bool doCall) => (bool success)
+transferData(address to, uint value, bytes data, bool doCall) => void
 ```
-Returns: bool, Success of the operation.
+
+No return value. Throws when not successful
 
 ### transferDataFrom
 
 Transfer  an allowed amount of tokens from one address to another.
 ```
-transferDataFrom(address from, address to, uint value, bytes data, bool doCall) => (bool success)
+transferDataFrom(address from, address to, uint value, bytes data, bool doCall) => void
 ```
-Returns: bool, Success of the operation.
-
-## DAO Methods
-
-In order for the DAO to work the deployer of the contract has to set the minimun votes required for every action that the DAO can execute, so once the contract is created the deployer has to use buildMinVotes() function to define the votes needed for every action type, after that once the contract status is correct the DAO can start.
+No return value. Throws when not successful
 
-Every DAO proposal will be an action type, depend on the action type the amounts of votes that will be required by the contract to execute the proposal. Every function on Ethereum has a signature, data and value.
-The signature is what the contract need to know which function execute, the data is the data that will be sent to this function and the value are the ethers that would be transferred on the function call.
-
-### Standard DAO functions
-
-This functions can be called only internally, in order to do that a token holder needs to create a proposal and reach the necessary votes to be executed.
+## Test
 
-```
-setBaseProposalFee(uint _baseProposalFee)
-setProposalBlocksWait(uint _proposalBlocksWait)
-sendEther(address to, uint amount)
-setStatus(uint newStatus)
-addDAOAction(address target, uint balanceNeeded, bytes4 signature)
-```
+* To run all tests: `npm test`
 
-### New Proposal
+* To run a specific test: `npm test -- test/Crowdsale.js`
 
-Creates a new proposal on the token, the token holder that creates the proposal needs to have the more than the fee charged for the proposal creation.
-```
-newProposal(address target, uint value, string description, uint agePerBlock, bytes4 signature, bytes actionData)
-```
+There are also two environment variables (`GEN_TESTS_QTY` and `GEN_TESTS_TIMEOUT`) that regulate the duration/depth of the property-based tests, so for example:
 
-### Vote
+    GEN_TESTS_QTY=50 GEN_TESTS_TIMEOUT=300 npm test
```

Comment:
i think you can use \`\`\`sh for a syntax highlighting, well, it is not that more beautiful in this case:

```sh
  GEN_TESTS_QTY=50 GEN_TESTS_TIMEOUT=300 npm test
```